metadata-db-app/src/cmr/db.clj

```diff
@@ -91,11 +92,18 @@
         (drop-user)
 
         (= "migrate" op)
-        (drift/run
-          (conj
-           (vec args)
-           "-c"
-           "config.mdb_migrate_config/app-migrate-config"))
+        ; drift looks for migration files within the user.directory, which is /app in service envs.
+        ; Dev dockerfile manually creates /app/cmr-files to store the unzipped cmr jar so that drift
+        ; can find the migration files correctly
+        ; we had to force method change in drift to set the correct path
+        (try
+          ;; trying non-local path to find drift migration files
+          (with-redefs [drift.core/user-directory (fn [] (new File (str (.getProperty (System/getProperties) "user.dir") "/cmr-files")))]
```

Comment:
is this for local writing or inside a docker image. I have no issue if it doing this in a docker image, but if it is local, it should go inside of .config or as defined by the ENV XDG_CONFIG_HOME

Reply:
This is for drift to use a specific path. I don't want to set this path along anywhere else

Reply:
but will this start creating directories on all our machines? It can not be /tmp?

Reply:
It shouldn't. It will find the pre-exiting cmr-files dir in the docker container.